### ISSUE TITLE
Remove index from CountBase

### ISSUE THREAD

**Issue by passabilities** [EdgeApp/baselet#4]
When I need to remove an element from an array fetched from `CountBase` there is no clear way to do this. I'm able to set an element to `null` in the array and then shift elements to the correct index but the `config` for a `CountBase` is setup to only increment the length of a bucket as seen [here](https://github.com/EdgeApp/baselet/blob/master/src/CountBase.ts#L70)

@bruscantini won't let me assign you to the issue for some reason

**Comment by paullinator** [comment]
You are correct and the specification for countbase does not allow deletion of elements. Doing so would incur a large performance hit. What is the condition where you would need to delete an element?

**Comment by passabilities** [comment]
Removing a UTXO

**Comment by paullinator** [comment]
Those should go in a rangebase not a countbase. Although admittedly we do not have a way of removing an item from the rangebase but we should add it
